compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -619,6 +628,8 @@ pub(crate) fn global_llvm_features(
             .features
             .split(',')
             .filter(|v| !v.is_empty() && backend_feature_name(sess, v).is_some())
+            // Drop +v8plus feature introduced in LLVM 20.
+            .filter(|v| *v != "+v8plus" || get_version() >= (20, 0, 0))
```

Comment:
This line here looks wrong... such cases should be handled in `to_llvm_feature`. And you did add that case there as well. So why did you also add it here? What makes this feature different from all the others that do not get a special case here?

Reply:
Ah, these are the LLVM feature names from the target, they don't go through `to_llvm_feature`, I see.